### DISCUSSION TITLE
Voice consistency Eleven Labs

### DISCUSSION THREAD

**Discussion by guiaugustinegri** [souzatharsis/podcastfy#156] [Q&A] [answered]
Hi guys, I have a question about voice consistency. When I see the app working by making requests in Eleven Labs, it sends a piece of text at a time, and then it re-generates the audio several times. When I do this directly in the Eleven Labs interface, it is possible to configure two parameters, Stability and Style Exaggeration. By setting the stability to high (more stable), we can make the re-generations sound very similar.

I am having trouble keeping the voices consistent. They change pitch and intonation, which has made them sound very artificial. Is there a way to set the stability in the configuration?

![eleven](https://github.com/user-attachments/assets/c020cf8a-7c3b-47a9-99f8-5214b8f30418)y.

**Comment by souzatharsis** [comment, marked as answer]
Not at the moment. But it should be straightforward to add that param.
Could you please elaborate on your use case + share the full set of desired
parameters and why?

<http://linkedin.com/in/tharsissouza>

**Reply by guiaugustinegri** [reply]
I believe it is for the main function, which is to build a podcast through content. In the OpenAI model, the standard audio regenerations are very similar, you can't tell and the audios are quite consistent and sound like the same person, but in the Elevenlabs models, especially the ones customized from human voice, the regenerations are quite different, they sound like different people, which ruins the experience of listening to the podcast.

Especially the "instant voice clone" type, which are smaller voice models in ElevenLabs, with each generation of the phrase, the results change a lot, sometimes the pitch, other times the intonation. It really gets weird.

Example: https://audio.com/guiherme-negri/audio/ex

Compare the same model 00:21 vs 01:35
